common/src/main/java/com/microsoft/identity/common/adal/internal/AuthenticationConstants.java

```diff
@@ -1429,6 +1438,8 @@ public enum API {
             MSAL_HELLO(MSAL_HELLO_PATH, null, VERSION_3),
             ACQUIRE_TOKEN_INTERACTIVE(MSAL_ACQUIRE_TOKEN_INTERACTIVE_PATH, null, VERSION_3),
             ACQUIRE_TOKEN_SILENT(MSAL_ACQUIRE_TOKEN_SILENT_PATH, null, VERSION_3),
+            FETCH_DCF_AUTH_RESULT(MSAL_FETCH_DCF_AUTH_RESULT_PATH, null, VERSION_3),
+            ACQUIRE_TOKEN_DCF(MSAL_ACQUIRE_TOKEN_DCF_PATH, null, VERSION_3),
```

Comment:
version_3 needs to be changed to the latest version. This is for msal version compatibility. Check GET_SSO_TOKEN(GET_SSO_TOKEN_PATH, null, VERSION_7) which has higher msal version than what you have here.  You may have to add a new version here as your changes are both in msal and broker